components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/IdentityOathEventListener.java

```diff
@@ -76,8 +79,8 @@ public boolean doPreDeleteUser(java.lang.String username,
 
     @Override
     public boolean doPostSetUserClaimValues(String userName, Map<String, String> claims, String profileName,
```

Comment:
Let's implement this in pre listener. Also need to implement doPreSetUserClaimValue()